### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
On the back of fixing security vulnerabilities in other sdx repos, removing unused and updating old dependencies seems like the prudent thing to do.  

To test this, fire it up in console and put a submission though.  Everything should work as normal. 
Additionally, the unit tests should still work (needed to update a bit of logging to get it to pass with the updated structlog)

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: app/response_processor.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def service_name(self, url=None):
             elif 'validate' in parts:
                 return 'SDX-VALIDATE'
         except AttributeError as e:
-            self.logger.error("No valid service name", exception=e)
+            self.logger.error("No valid service name", exception=str(e))
```
Comment (benhowes) [inline app/response_processor.py]: Better to pass `exc_info` https://docs.python.org/3.6/library/logging.html#logging.debug. i.e. `self.logger.error("No valid service name", exc_info=e)`. Unless I've missed something and this is a special logger?

Comment (insacuri) [reply app/response_processor.py]: I'm glad you mentioned that actually.  Looking at it now, it should just be logger.exception("...") as that adds the exception information in there for you.
We've done in in other places and it's worked, so we might as well do it here right?  No point in defining fields if we can get it for free instead!

Comment (nealedj) [reply app/response_processor.py]: Yeah if you don't need to write a custom error message then probs best to just call `logger.exception`

Comment (benhowes) [reply app/response_processor.py]: Oh, sure, if that log level is fine!

**Pull request merged**